e2e/ios/Podfile

```diff
@@ -13,30 +13,6 @@ install! 'cocoapods',
 
```

Comment:
I changed this file based on changes suggested by [react-native upgrade helper](https://react-native-community.github.io/upgrade-helper/) for react-native 0.73 -> 0.74